subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsTask.kt

```diff
@@ -121,6 +121,13 @@ abstract class InstrumentationTestsTask @Inject constructor(
     @OutputDirectory
     val output: DirectoryProperty = objects.directoryProperty()
 
+    @Internal
+    private val verdictFile = objects.fileProperty().convention(output.file("verdict.json"))
+
+    @get:Internal
+    override val verdict: String
+        get() = verdictFile.asFile.get().readText()
```

Comment:
Is it a final decision? I thought about links to reports

Reply:
I'm thinking about this implementation like about prototype
And we need to discuss improvements and then create tasks for them or I could implement them

Reply:
Ok